dm/loader/checkpoint.go

```diff
@@ -116,6 +128,7 @@ func newRemoteCheckPoint(tctx *tcontext.Context, cfg *config.SubTaskConfig, id s
 		logger:         tctx.L().WithFields(zap.String("component", "remote checkpoint")),
 	}
 	cp.restoringFiles.pos = make(map[string]map[string]FilePosSet)
+	rollbackHolder.Add(fr.FuncRollback{Name: "CloseRemoteCheckPoint", Fn: cp.Close})
 
 	err = cp.prepare(tctx)
```

Comment:
how about close DB in func `cp.prepre`, so we need't and rollback func anymore

Reply:
A uniform rollback can handle all problems like this in the future, i had done it with close in prepare, but i think rollback is better.